apps/mull-api/src/app/event/event.service.ts

```diff
@@ -27,6 +27,13 @@ export class EventService {
     return this.findOne(eventInput.id);
   }
 
+  async addParticipant(eventId: number, userId: number) {
+    const user = new User(userId);
+    const event = await this.eventRepository.findOne(eventId, { relations: ['participants'] });
+    event.participants.push(user);
```

Comment:
Another question, what happen if the user id already exists in the array?

Reply:
I also had the same question when I was writing this code. The ORM handles it for us, it won't add duplicate users as participants in the database.

Reply:
Oh alright, thank you!